.scripts/process_build.sh

```diff
@@ -0,0 +1,175 @@
+  rm src/ocaml-output/fstar/HelloFStarOutput.log
+fi
+if [[ -f src/ocaml-output/fstar/AllExamples.log ]]; then
+  rm src/ocaml-output/fstar/AllExamples.log
+fi
+
+# Need this to get back after unzip things
+ORIG_PWD=$PWD
+
+echo "*** Make package ***"
+cd src/ocaml-output
+make package
+
+# 'make package' makes the package using the major version from version.txt. This script is a weekly process to make minor versions so use timestamp in file name instead of major version
+echo "*** Unzip and verify the Package  ***"
+TIME_STAMP=$(date +%s)
+
+TYPE="_Windows_x64.zip"
+MAJOR_ZIP_FILE=fstar_$CURRENT_VERSION$TYPE
+if [[ -f $MAJOR_ZIP_FILE ]]; then
+  MINOR_ZIP_FILE=fstar_$TIME_STAMP$TYPE
+  cp $MAJOR_ZIP_FILE $MINOR_ZIP_FILE
+  unzip -o $MAJOR_ZIP_FILE
+else
+  TYPE="_Linux_x86_64.tar.gz"
+  MAJOR_TAR_FILE=fstar_$CURRENT_VERSION$TYPE
+  if [[ -f $MAJOR_TAR_FILE ]]; then
+    MINOR_TAR_FILE=fstar_$TIME_STAMP$TYPE
+    cp $MAJOR_TAR_FILE $MINOR_TAR_FILE
+    tar -x -f $MAJOR_TAR_FILE
+  else
+    echo -e "* ${RED}FAIL!${NC} src/ocaml-output/make package did not create "$MAJOR_ZIP_FILE" or "$MAJOR_TAR_FILE
+    exit 1
+  fi
+fi
+
+echo "*** Make the examples ***"
+cd fstar
+make -C examples/micro-benchmarks 1> MicroBenchMarkOutput.log
+make -C examples/hello ocaml 1> HelloOcamlOutput.log
+make -C examples/hello fs 1> HelloFStarOutput.log
+make -j6 -C examples 1> AllExamples.log
+
+echo "*** Verify the examples ***"
+echo "-- Verify Micro-benchmarks -- should output 'Success:' *"
+if ! egrep 'Success:' MicroBenchMarkOutput.log; then
+  echo -e "* ${RED}FAIL!${NC} for examples/micro-benchmarks - Success: was not found in MicroBenchMarkOutput.log"
+  exit 1
+else
+  echo -e "* ${GREEN}PASSED!${NC} for examples/micro-benchmarks"
+fi
+
+echo "-- Verify hello ocaml -- should output Hello F*! *"
+if ! egrep 'F*!' HelloFStarOutput.log; then
+  echo -e "* ${RED}FAIL!${NC} for examples/hello ocaml - F*! was not found in HelloOcamlOutput.log"
+  exit 1
+else
+  echo -e "* ${GREEN}PASSED!${NC} for examples/hello ocaml"
+fi
+
+echo "-- Verify hello fs -- should output Hello F*!"
+if ! egrep 'F*!' HelloFStarOutput.log; then
+  echo -e "* ${RED}FAIL!${NC} for examples/hello fs - F*! was not found in HelloFStarOutput.log"
+  exit 1
+else
+  echo -e "* ${GREEN}PASSED!${NC} for examples/hello fs"
+fi
+
+echo "-- Verify all examples -- Look for Success:"
+if ! egrep 'Success:' AllExamples.log; then
+  echo -e "* ${RED}FAIL!${NC} for all examples - Success: was not found in AllExamples.log"
+  exit 1
+else
+  echo -e "* ${GREEN}PASSED!${NC} for all examples"
+fi
+
+# Got to this point, so know it passed - copy minor version out
+echo "*** Upload the minor version of the package. Will only keep the most recent 4 packages ***"
+cd $ORIG_PWD
+BN_BINARYSPATH_ROOT=~/binaries
+BN_BINARYSPATH=$BN_BINARYSPATH_ROOT/weekly
+BN_FILESTOKEEP=4
+
+if [[ ! -d $BN_BINARYSPATH_ROOT ]]; then
+  cd ~
+  git clone https://github.com/FStarLang/binaries.git
```

Comment:
this is an https url... which by default won't use ssh authentication; do want `git@github.com:/FStarLang/...` instead?